apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -63,27 +63,45 @@ export default function GlobalStyle () {
                   position: relative;
                   bottom: 4px;
                 }
+                
+                & .ant-checkbox-inner {
+                  background-color: inherit;
+                  border: 1px solid ${colors.inputBorderGrey};
+                  border-radius: 4px;
+                             
+                  width: 24px;
+                  height: 24px;
+                  
+                  &:focus {
+                    background-color: inherit;
+                  }
+                  
+                  &::after {
+                    left: 36%;
+                    width: 7px;
+                    height: 12px;
+                  }
+                }
+              }
+              
+              .ant-checkbox-wrapper:hover .ant-checkbox-inner, .ant-checkbox:hover .ant-checkbox-inner,
+              .ant-checkbox-input:focus + .ant-checkbox-inner {
+                border-color: ${colors.inputBorderHover};
```

Comment:
`import { colors  } from "@condo/ui/colors"`

And try to avoid global styles modifying